docs/how-to-deploy-a-bundle.md

```diff
@@ -0,0 +1,74 @@
+# Creating and Installing a Custom Build
+
+You can build a cluster-logging-operator directly from source code,
+and create a bundle image that will allow you to deploy it to your cluster.
+
+You need to create two images:
+
+* operator image: Contains your build of the operator.
+* bundle image: Contains instructions for OLM to install you operator.
+
+## Log-in to Image Repositories
+
+You need a public image registry, for example `quay.io/somebody`, to _write_ your operator and bundle images.
+
+**NOTE**: The images you create must be _public_ so your cluster will be able to pull them.
+For example, on `quay.io` you need to mark images as public in the "organization settings".
+
+This document will use `$REGISTRY` to refer to your image registry.
+
+You also need to log in to the red Hat registry for related images.
+Use your Red Hat customer portal username and password.
+See https://access.redhat.com/RegistryAuthentication for more.
+
+``` bash
+podman login $REGISTRY
+podman login registry.redhat.io
+```
+
+Note: this document uses `podman` but you can substitute `docker` instead.
+
+## Build and Push Operator and Bundle Images
+
+Clone the cluster-logging-operator repository:
+
+``` bash
+git clone http://github.com/openshift/cluster-logging-operator
+cd cluster-logging-operator
+```
+
+Build and push the operator image:
+
+``` bash
+make image IMAGE_TAG=$REGISTRY/my-cluster-logging-operator:latest
+podman push $REGISTRY/my-cluster-logging-operator:latest
+```
+
+Build and push the bundle image:
+
+``` bash
+make bundle
+podman build -t $REGISTRY/my-cluster-logging-bundle:latest -f bundle.Dockerfile
+podman push $REGISTRY/my-cluster-logging-bundle:latest
+```
+
+# Installing the Bundle Image
+
+You need `operator-sdk` which you can get here: https://sdk.operatorframework.io/docs/installation
+
+**NOTE**: you need `operator-sdk.v1.22.2` or greater.
+
+To install the image:
+```
+oc create namespace openshift-logging
+operator-sdk run bundle -n openshift-logging $REGISTRY/my-cluster-logging-bundle:latest
```

Comment:
This may fail based on my experiment.  operator-sdk advised me to add an additional switch: https://coreos.slack.com/archives/C3VS0LV41/p1658515952708889 
```
The workaround for now is to use the --index-image flag with run bundle to pin to a
specific (not yet broken) opm image. e.g. --index-image=[quay.io/operator-framework/opm:v1.23.0]
(http://quay.io/operator-framework/opm:v1.23.0)
```

Reply:
I think this is fixed in operator-sdk-v1.22.2 and I updated the version in the repo. If you still have trouble we'll add the flag.